ramon/rubocop 2

helloramon opened the pull request:
Continuing Rubocop fixes. Mostly whitespace errors, double-quotations to singles, and created a .rubocop.yml and turned off the class comment nag via (Documentation: Enabled: false). 

Also found extensive info on how to selectively pick which Rubocop offenses are allowed here: https://github.com/bbatsov/rubocop#disabling-cops-within-source-code If you generate a form and add an inheritance in the .rubocop.yml file, all the current errors will be turned off and you can be selective as to which ones to turn on. I didn't generate one because I don't know which ones we want on/off.

jcrides commented:
I think we should hold off on merging this one until we talk on Thursday. It has changes to the ruby 1.9 hash syntax  (symbol: 'value' vs :symbol => 'value'). I think we all need to agree we want to switch to this before we make these changes. Basically saying that just because rubocop says it should be this way doesn't mean we have to do it, as long as we pick one way and do it consistently.

helloramon commented:
Thanks Jeremy, I think I ended up turning into a Rubocop sheep and just started going through everything I could. xD

I reverted the hash syntax changes and added to the .rubocop.yml so if we want to keep the hashrocket and ignore those "fixes", we can just set the `Style/HashSyntax:` to `false`.
